Apps/Sandcastle/index.html

```diff
@@ -21,6 +21,7 @@
     <script src="jsHintOptions.js"></script>
     <script src="gallery/gallery-index.js"></script>
     <script src="CesiumSandcastle.js"></script>
+    <script src="../../Source/Cesium.js"></script>
```

Comment:
I can try testing it. Do you mean seeing if its slower just by eye?

Does seem like a lot just to retrieve the version number though. Is there anywhere else we can get it that number?